projects/storefrontlib/src/cms-components/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.ts

```diff
@@ -23,7 +30,12 @@ export class AddedToCartDialogComponent implements OnInit {
   entry$: Observable<OrderEntry>;
   cart$: Observable<Cart>;
   loaded$: Observable<boolean>;
+  addedEntryWasMerged$: Observable<boolean>;
+  /**
+   * @deprecated since 3.0, set numberOfEntriesBeforeAdd instead
```

Comment:
Created ticket to remove it in the next major. https://github.com/SAP/spartacus/issues/9288